cmd/git-sync/main.go

```diff
@@ -527,7 +528,7 @@ func revIsHash(ctx context.Context, rev, gitRoot string) (bool, error) {
 func syncRepo(ctx context.Context, repo, branch, rev string, depth int, gitRoot, dest string) (bool, error) {
 	target := path.Join(gitRoot, dest)
 	gitRepoPath := path.Join(target, ".git")
-	hash := rev
```

Comment:
rationale: `switch` has `default` case where hash variable is assigned, so this assignement has no effect.

Reply:
good one